src/client/components/RoomListOverview/RoomListOverview.css

```
@@ -2,4 +2,14 @@
   display: flex;
   border: 2px solid #555;
   justify-content: center;
+  overflow-x: auto;
+  min-width: 30px;
+}
+
+.roomlist-overview::-webkit-scrollbar {
+  width: 0;
+}
+
+.pic-and-title {
```

Always use scoping in components.

```suggestion
.roomlist-overview .pic-and-title {
```